config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
```

Comment:
Do we need this since we're not using action cable?

Reply:
Probably not, I'm happy to drop it.